arches_modular_reports/src/arches_modular_reports/ModularReport/types.ts

```diff
@@ -85,7 +85,7 @@ export interface NodeData {
 export type NodegroupData = TileData | TileData[];
 
 export interface AliasedData {
-    [key: string]: NodeData | NodegroupData | null;
+    [key: string]: NodeData | NodegroupData;
 }
```

Comment:
This can be null if there are no tiles for a nodegroup alias. We may not be seeing it in the data tree because we're always requesting blank tiles at every nodegroup, but there's no reason to expect that to be the case always.